### PR TITLE
Text Analyzer

### PR DESCRIPTION
**Pull request by khushi-purwar**
#94 has been solved. @chicken-biryani and @Ayushparikh-code  please check my PR.

SS1

### CODE REVIEW

File: Text Analyzer/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            </div>
+
+        </div>
+    </div>
+
+    <script src="script.js"></script>
+</body>
+</html>
+
+
```
Comment (chicken-biryani) [inline Text Analyzer/index.html:54]: Beautify your HTML code.

USe the link https://codebeautify.org/htmlviewer

File: Text Analyzer/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Text Anaylzer
+
+## Description
+The Text Analyzer analyze the text entered by the user. After clicking on the process text, it will give information such as word count, character count, number of sentences, number of spaces, number of punctuators used in the text.
+
+## Stacks Used
+* JavaScript
+* HTML
+* CSS
+
```
Comment (chicken-biryani) [inline Text Analyzer/README.md:10]: Add "How to set - up Project" Section over here.

File: Text Analyzer/script.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  return senCount ? senCount.length : 0;   
+   
+}
+
+function puncCount(text){
+    const regex= /[.,?;:'"!-(){}]/g;
+    let puntuationCount = text.match(regex);
+    return puntuationCount ? puntuationCount.length: 0;
+}
+
```
Comment (chicken-biryani) [inline Text Analyzer/script.js:51]: Use JS beautifier.
Use link -- https://codebeautify.org/jsviewer

File: Text Analyzer/style.css [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    gap: 5px;
+}
+.text-entry{
+    border-right: none;
+    border-bottom: 3px dotted blue;
+   
+}
+}
+
+
```
Comment (chicken-biryani) [inline Text Analyzer/style.css:133]: Remove extra lines towards the end.

Use a css beautifier.

**Comment by khushi-purwar** [comment]
All the changes have been made. Please check @chicken-biryani

**Review by Ayushparikh-code — APPROVED** [review]
LGTM

**Pull request closed without merging**